packages/editor/src/store/actions.js

```diff
@@ -536,49 +537,6 @@ export function stopTyping() {
 	};
 }
 
-/**
- * Returns an action object used to create a notice.
- *
- * @param {string}    status  The notice status.
- * @param {WPElement} content The notice content.
- * @param {?Object}   options The notice options.  Available options:
- *                              `id` (string; default auto-generated)
- *                              `isDismissible` (boolean; default `true`).
- *
- * @return {Object} Action object.
- */
-export function createNotice( status, content, options = {} ) {
-	const {
-		id = uuid(),
-		isDismissible = true,
-		spokenMessage,
-	} = options;
-	return {
-		type: 'CREATE_NOTICE',
-		notice: {
-			id,
-			status,
-			content,
-			isDismissible,
-			spokenMessage,
```

Comment:
This code assumes that from now on `spokenMessage` is always equal to `content` - is it correct?

Reply:
I believe so, in that previously we'd allow `spokenMessage` to be of type `element`, which is no longer supported (must be string).

Reply:
Yeah right. It makes stnyyo consolidate now.